hunts/views.py

```diff
@@ -74,6 +74,8 @@ def post(self, request, pk):
 
         if form.is_valid():
             name = form.cleaned_data["name"]
+            if len(name) > 20:
+                name = name[:20]
```

Comment:
I think we should make this 80, which is what we need to fix the 

```
Exception Value: value too long for type character varying(80)
```

error @maximized saw. As for the scrolling (table being too wide), I think that's mainly due to the search boxes at the bottom of the table being too wide -- see my comment here: https://github.com/cardinalitypuzzles/smallboard/pull/103#issuecomment-567924232

Reply:
I think the discrepancy between what you are seeing and what Max saw is due to the fact that I disabled horizontal table scrolling in https://github.com/cardinalitypuzzles/smallboard/pull/82. 

I think you're right in general -- we should only truncate if we absolutely have to. I've changed my code to reflect this.